test/dynamo/test_subclasses.py

```diff
@@ -128,6 +128,26 @@ def f(x):
         self.assertEqual(cnt.frame_count, 3)
         self.assertEqual(cnt.op_count, 3)
 
+    def test_compile_with_functional_tensor_wrapper(self):
+        x = torch.randn([3, 4])
+        cnt = torch._dynamo.testing.CompileCounter()
+
+        @torch.compile(backend=cnt, fullgraph=True)
+        def f(x):
+            return torch.sin(x)
```

Comment:
- Can we put some mutable operations into the graph and then do an expect test on the graph that sent to the backend? I expect we will see all the mutable operations in the graph
- Can we assert that the sample inputs passed to the backend are actually FunctionalTensorWrapper?
- Could we check that torch.compile(backend='eager', fullgraph=True) works? (this is the case that we care about for torch.cond)

Reply:
Good idea!